go-cli/internal/commands/start.go

```
@@ -0,0 +1,161 @@
+		"tipi-redis",
+		"tipi-worker",
+		"tipi-dashboard",
+		// New naming
+		"runtipi",
+		"runtipi-reverse-proxy",
+		"runtipi-db",
+		"runtipi-redis",
+	}
+
+	for _, container := range containerNames {
+		exec.Command("docker", "stop", container).Run()
```

golangci-lint (1.64.8)

Error return value of `(*os/exec.Cmd).Run` is not checked

(errcheck)